google/resource_compute_network_endpoint_group.go

```diff
@@ -80,8 +80,8 @@ you create the resource.`,
 				Type:         schema.TypeString,
 				Optional:     true,
 				ForceNew:     true,
-				ValidateFunc: validation.StringInSlice([]string{"GCE_VM_IP_PORT", ""}, false),
-				Description:  `Type of network endpoints in this network endpoint group. Default value: "GCE_VM_IP_PORT" Possible values: ["GCE_VM_IP_PORT"]`,
+				ValidateFunc: validation.StringInSlice([]string{"GCE_VM_IP", "GCE_VM_IP_PORT", ""}, false),
+				Description:  `Type of network endpoints in this network endpoint group. Default value: "GCE_VM_IP_PORT" Possible values: ["GCE_VM_IP", "GCE_VM_IP_PORT"]`,
```

Comment:
Updating the docs here & in the related markdown file is great, but the list should match what's in the [API docs](https://cloud.google.com/compute/docs/reference/rest/beta/networkEndpointGroups)

Reply:
This list should not match what's in the API docs; load balancers are weird, and this resource only corresponds to one type of load balancer.